### ISSUE TITLE
Release 3.1.0

### ISSUE THREAD

**Issue by lholmquist** [cloudevents/sdk-javascript#300]
3.0.1 was released the 29th of August, but we also forgot to update the browser version to the new name.   The commits below include that update.   Should we release a new patch release soon with that change?  I don't think there are any commits below that need a minor bump.  I can see an argument for a7e0aa0 as a minor bump,  but I don't think it needs to be

```
git log v3.0.1..origin/master --oneline

a7e0aa0 lib: expose constants in a more intuitive way (#299)
bc81242 build(deps): bump elliptic from 6.5.2 to 6.5.3 (#290)
31c2005 refactor: no import star (#297)
48d182b fix: update browser name to cloudevents. (#292)
763838c chore: Update examples to use latest sdk changes (#282)
3fab5f2 docs: update badge name (#289)
ed9ea95 fix: ensure that data encoded as base64 is parsed as an object (#285)
e219a30 chore: Update readme with correct Receiver usage (#287)
```

**Comment by lance** [comment]
> 3.0.1 was released the 29th of August

That's some powerful time machine work you've got there! :laughing:

**Comment by lance** [comment]
+1 for a new release

**Comment by lholmquist** [comment]
> That's some powerful time machine work you've got there! laughing

:facepalm:

**Comment by lance** [comment]
@lholmquist thinking a bit more about this, I think maybe this warrants a 3.1.0 release. The change with `CONSTANTS` being exported is a breaking change, and will actually break an automated upgrade.

**Comment by lholmquist** [comment]
@lance that is fine.

**Comment by lholmquist** [comment]
lets make sure this makes it in too, https://github.com/cloudevents/sdk-javascript/pull/302

**Comment by lholmquist** [comment]
Updating the list of commits:

```
git log v3.0.1..origin/master --oneline

a5249de chore:  fix promise tests to break the build when they fail (#305)
ad0c434 fix: Add Correct Headers to emitted Binary Event (#302)
a7e0aa0 lib: expose constants in a more intuitive way (#299)
bc81242 build(deps): bump elliptic from 6.5.2 to 6.5.3 (#290)
31c2005 refactor: no import star (#297)
48d182b fix: update browser name to cloudevents. (#292)
763838c chore: Update examples to use latest sdk changes (#282)
3fab5f2 docs: update badge name (#289)
ed9ea95 fix: ensure that data encoded as base64 is parsed as an object (#285)
e219a30 chore: Update readme with correct Receiver usage (#287)
```

**Comment by lholmquist** [comment]
So doing a `npm run release -- --dry-run`  the version that was created, was 3.0.2 based on the commits.

@lance @grant I think we might need to force a 3.1.0 if we feel #299 makes it a minor bump instead of a patch bump

**Comment by lance** [comment]
@lholmquist I think we should... You may know how already, but it's pretty easy to force a version number.

```
npm run release -- --release-as 3.2.0
```

**Comment by lholmquist** [comment]
yup,  i am familiar

**Comment by lholmquist** [comment]
There have been a couple commits that came in today, which were doc and some process related things,  3.1.0 is now looking like:


08bf15d  docs: Release Guidelines (#306)
f3cc2b4 chore: update the release script to signoff the commit (#307)
a5249de chore:  fix promise tests to break the build when they fail (#305)
ad0c434 fix: Add Correct Headers to emitted Binary Event (#302)
a7e0aa0 lib: expose constants in a more intuitive way (#299)
bc81242 build(deps): bump elliptic from 6.5.2 to 6.5.3 (#290)
31c2005 refactor: no import star (#297)
48d182b fix: update browser name to cloudevents. (#292)
763838c chore: Update examples to use latest sdk changes (#282)
3fab5f2 docs: update badge name (#289)
ed9ea95 fix: ensure that data encoded as base64 is parsed as an object (#285)
e219a30 chore: Update readme with correct Receiver usage (#287)

**Comment by lholmquist** [comment]
Released now as 3.1.0, https://www.npmjs.com/package/cloudevents/v/3.1.0
